src/component/annotator/AnnotatorTermsSelector.tsx

```diff
@@ -33,17 +33,26 @@ const AnnotatorTermsSelector: React.FC<AnnotatorTermsSelectorProps> = ({
   const { annotatorTerms, vocabulary, terminalStates } = useSelector(
     (state: TermItState) => state
   );
+  const [options, setOptions] = useState([] as Term[]);
```

Comment:
Do you think it is necessary to store the options in state? They do not change by any actions within the component. If it is for caching reasons, wouldn't memoization make more sense?